README.md

```diff
@@ -1,24 +1,169 @@
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 Motorcycle Booking - RoR <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+The Motorcycle Booking App has the following features :
 
-* Deployment instructions
+-----------------------------------------------------------------------------------------------------------------------
+- Created a new Rails API app called 'motorcycle-booking'.
+- Set up the repository on GitHub and used Gitflow.
+- Used Postgres as database.
+- Used Rails to create backend API.
+- Set up project as two separate apps. (This one is backend)
+- Created table to store data.
+- Created API endpoints to interact with the database.
 
-* ...
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Data Base</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Rails</li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Use of endpoints]**
+- **[Perform CURD operation on database]**
+- **[API Usage]**
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+## React Project Link <a name="react-link"></a>
+https://github.com/jonas-45/motorcyle-booking
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+Will be available shortly...
+
+
+### Video Demo Link <a name="live-demo-link"></a>
+Will be available shortly...
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- You'll need any code editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git@github.com:jonas-45/motorcyle-booking.git
+  cd motorcyle-booking
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  bundle install
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+## 👥 Authors <a name="authors"></a>
+
+🕵🏽‍♀️ **Ahmad Saeed**
+
+- Github: [GitHub](https://github.com/ehmaddd/)
+- Twitter: [Twitter](https://twitter.com/ehmaddd_pk)
+- Linkedin: [LinkedIn](https://www.linkedin.com/in/ehmaddd/)
+
```

Comment:
Hi @ehmaddd,
you have done a good job making your Readme file look professional, but there is one issue you still have to work on before you move to the next step.
Kindly add other team members as authors since it's a group project.

Reply:
It's done as instructed